safe/processors/__init__.py

```diff
@@ -17,10 +17,15 @@
 # |--- size
 # |   |--- size rate  disabled in V4.0, ET 13/02/17
 # |   `--- productivity
+# |   `--- production cost
+# |   `--- production value
 # |--- distance
 # |--- bearing angle
 # |   |--- cardinality
 # |--- affected
+# |   `--- affected productivity (require productivity)
+# |   `--- affected production cost (needs production cost)
+# |   `--- affected production value (require production value)
```

Comment:
Ouch, it becomes an internal nested tree now ...

Reply:
Yes, sorry for that. Actually it can be a tree, since the affected productivity needs two inputs from two different post processors.

Reply:
No worries, I understood ;-)